src/common/types/profile-derived.js

```diff
@@ -1,14 +1,23 @@
 // @flow
-export type IndexIntoFuncStack = number;
+import type { Milliseconds } from './units';
+
+export type IndexIntoFuncStackTable = number;
 
 export type FuncStackTable = {
   prefix: Int32Array,
   func: Int32Array,
   depth: number[],
   length: number,
-}
+};
```

Comment:
Looks like there is, although I'm not sure the flowtype rules are always working, since my lazy style habits aren't always fixed. I've been a bit more particular since I noticed that.

https://github.com/gajus/eslint-plugin-flowtype#semi